### ISSUE TITLE
Alias Literal in Literal declaration

### ISSUE THREAD

**Issue by Morikko** [microsoft/pyright#757]
**Describe the bug**
`Type arguments for "Literal" must be None, a literal value (int, bool, str, or bytes), or an enum value` but not a Literal alias like describe here: https://www.python.org/dev/peps/pep-0586/#legal-parameters-for-literal-at-type-check-time

> Literal may also be parameterized by other literal types, or type aliases to other literal types. For example, the following is legal

**To Reproduce**
```python
ReadOnlyMode         = Literal["r", "r+"]
WriteAndTruncateMode = Literal["w", "w+", "wt", "w+t"]
WriteNoTruncateMode  = Literal["r+", "r+t"]
AppendMode           = Literal["a", "a+", "at", "a+t"]

AllModes = Literal[ReadOnlyMode, WriteAndTruncateMode,
                   WriteNoTruncateMode, AppendMode]
```

<img width="894" alt="Capture d’écran 2020-06-24 à 14 56 27" src="https://user-images.githubusercontent.com/7831572/85560421-eb40de00-b62a-11ea-803d-5e194525d142.png">

**Expected behavior**
The type of `AllModes` should be `Literal["r", "r+", "w", "w+", "wt", "w+t", "r+", "r+t", "a", "a+", "at", "a+t"]`

**VS Code extension or command-line**
Pyright 1.1.45 is running as a VSCode extension.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next published version of Pyright.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.46, which I just published.
